JokeRandomizer/JokeRandomizer/ViewModel/JokesViewModel.swift

```diff
@@ -7,46 +7,54 @@
 
 
 import Foundation
+import Combine
 
-class JokesViewModel: ObservableObject {
```

Comment:
use a protocol to make an abstract class and inherit it in this class

Reply:
still working for the closures.